apk-generator/scripts/email.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+echo "Preparing to send the email"
+MPATH=$1
+EMAIL=$2
+mutt -s "Test mail" $EMAIL -a $MPATH/open-event-android/android/app/build/outputs/apk/app-googleplay-debug.apk < /var/www/mail_message.txt
+#mutt -s "test" manasbagula@gmail.com -a /root/Final/clone.sh < /root/mail_message.txt
```

Comment:
@mananwason There seems to be a strange email id associated with the comments in this bash file.
Please delete it.